tensorflow_datasets/text/definite_pronoun_resolution.py

```diff
@@ -49,11 +49,18 @@
 
 class DefinitePronounResolution(tfds.core.GeneratorBasedBuilder):
   """The Definite Pronoun Resolution Dataset."""
+  # Version history:
+  # 2.0.0: S3 with new hashing function (different shuffle).
+  # 1.0.0: S3 (new shuffling, sharding and slicing mechanism).
   BUILDER_CONFIGS = [
       tfds.core.BuilderConfig(
           name='plain_text',
           version=tfds.core.Version(
               '0.0.1', experiments={tfds.core.Experiment.S3: False}),
+          supported_versions=[
+            tfds.core.Version("2.0.0"),
+            tfds.core.Version("1.0.0"),
```

Comment:
You should only add a single new version, as only the new hash function is supported now.